unit_test/test_spdm_responder/csr.c

```diff
@@ -11,27 +11,21 @@
 
 #define LIBSPDM_MAX_CSR_SIZE 0x1000
 
+#define LIBSPDM_MAX_REQ_INFO_BUFFER_SIZE 200

```

Comment:
why 200?

Reply:
Thanks. 
I have referenced:
https://github.com/Mbed-TLS/mbedtls/blob/3048c8c90654eb116a6b17c0d2d27c3ccbe6782c/programs/x509/cert_req.c#L119-L129*/
to change the size to 4096.